.github/pipeline-descriptor.yml

```diff
@@ -0,0 +1,18 @@
+github:
+  username: ${{ secrets.GITHUB_USERNAME }}
+  token:    ${{ secrets.GITHUB_TOKEN }}
+
+codeowners:
+- path:  "*"
+  owner: "@buildpacks/buildpack-authors-tooling-maintainers"
+
+package:
+  repository:     docker.io/buildpacks/profile
+  register:       true
+  registry_token: ${{ secrets.GITHUB_TOKEN }}
+
+docker_credentials:
```

Comment:
Could we just publish to ghcr?

Reply:
Follow up from the BAT team meeting. We discussed and I believe the decision was to go ahead with using Docker Hub. We have other resources published there, so it makes sense to publish buildpack images there as well.